Gemfile

```diff
@@ -53,6 +53,8 @@ gem 'bootsnap', require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'ffi'
+  gem 'rails-controller-testing'
   gem 'rspec-rails'
 end
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update your `Getting Started` section in your `README.md` file by adding the `Run Tests` sub-section including the command of how to run your tests. This will make your documentation be more professional and descriptive

Reply:
Thanks for the suggestions @tresorsawasawa. I added the `Run Tests` sub-section in `README.md`